docs/core/porting/third-party-deps.md

```diff
@@ -104,6 +104,12 @@ To suppress the warning by editing the project file, find the `PackageReference`
 
 For more information on how to suppress compiler warnings in Visual Studio, see [Suppressing warnings for NuGet packages](/visualstudio/ide/how-to-suppress-compiler-warnings#suppressing-warnings-for-nuget-packages).
 
+### Port your packages to `PackageReference`
+
+.NET Core uses `PackageReference` to specify package dependencies. If you are using `packages.config` to specify your packages, you will need to convert over to `PackageReference`.
```

Comment:
`PackageReference` -> might be good to specify what this is and where, such as 
.NET Core uses `PackageReference` entries in the project file to specify package dependencies

link here perhaps for more info?
https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files